tests/calculation.test.js

```diff
@@ -0,0 +1,61 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictequals = require('../calculation')
+
+describe('calculation', () => {
+  describe('strictequals', () => {
+    it('returns true if the values are the same and does not type coerce', () => {
```

Comment:
Careful to always match your description to the actual test. Here you say it should return true but then assert that it returns false.